webapp/src/main/java/com/box/l10n/mojito/service/ai/LLMPromptService.java

```diff
@@ -74,11 +78,18 @@ public Long createPrompt(AIPromptCreateRequest AIPromptCreateRequest) {
     aiPromptRepository.save(aiPrompt);
     logger.debug("Created prompt with id: {}", aiPrompt.getId());
 
-    RepositoryLocaleAIPrompt repositoryLocaleAIPrompt = new RepositoryLocaleAIPrompt();
-    repositoryLocaleAIPrompt.setRepository(repository);
-    repositoryLocaleAIPrompt.setAiPrompt(aiPrompt);
-    repositoryLocaleAIPromptRepository.save(repositoryLocaleAIPrompt);
-    logger.debug("Created repository prompt with id: {}", repositoryLocaleAIPrompt.getId());
+    if (repository != null) {
+      RepositoryLocaleAIPrompt repositoryLocaleAIPrompt =
+          aiPromptType.getName().equals(PromptType.TRANSLATION.name())
```

Comment:
Do we want a case sensitive equals comparison here?

Reply:
The aiPromptType value will be retrieved from the DB, it uses the value of the prompt type enum so it should always match the name of the PromptType enum